docs/src/main/docs/microprofile/02_server-configuration.adoc

```diff
@@ -57,23 +58,29 @@ ssl:
     keystore-passphrase: "abcd"
 ----
 
+== Configure "admin" port
+For this example, we will use a `yaml` file. We configure a port 8011 that would server health and metrics endpoints,
+and then configure health and metrics to use it.
 
-For Helidon config, the default file is `application.yaml` on the classpath
- (e.g. src/main/resources/application.yaml).
+Alternative ports can be used by Server extensions that support it. This support is currently limited to MP Health and
```

Comment:
I think it would preferred to define the notion of an  "admin" port at the beginning of the section.
Also use admin or alternative, but not both terms.

Reply:
Reworded the whole section